src/QuantumSdk/Sdk/Sdk.props

```diff
@@ -0,0 +1,52 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Work around issues in MSBuild versions prior to 16.0. -->
+  <PropertyGroup>
+    <MSBuildAllProjects Condition=" '$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0' ">
+      $(MSBuildAllProjects);$(MSBuildThisFileFullPath)
+    </MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)../DefaultItems/DefaultItems.props"/>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk"/>
+  <Import Project="$(MSBuildThisFileDirectory)../ProjectSystem/ProjectSystem.props"/>
+
+
+  <!-- The following item groups indeed needs to live in this file to ensure 
+  that they evaluated before the modifications in the project file are applied. -->
+
+  <!-- Q# source files included by default. -->
+  <ItemGroup>
+    <QsharpCompile Include="**/*.qs" Exclude="obj/**" />
+  </ItemGroup>
+
+  <!-- Q# package references included by default. -->
+  <ItemGroup>
+    <!-- Packages and libraries included for all execution targets. -->
+    <PackageReference Condition="$(IncludeQsharpCorePackages)" Include="Microsoft.Quantum.QSharp.Core" Version="0.10.1912.2604-alpha" />
+    <PackageReference Condition="$(IncludeQsharpCorePackages)" Include="Microsoft.Quantum.Standard" Version="0.10.1912.2603-alpha" />
+    <!-- Packages included for specific execution targets. -->
+    <PackageReference Condition="'$(ResolvedQsharpExecutionTarget)' == 'SimulatorBackend'" Include="Microsoft.Quantum.Simulators" Version="0.10.1912.2604-alpha" IsImplicitlyDefined="$(CsharpGeneration)" />
+  </ItemGroup>
+
+  <!-- Packages containing rewrite steps that are executed during compilation. -->
+  <ItemGroup>
+    <PackageReference Condition="$(CsharpGeneration)" Include="Microsoft.Quantum.Runtime.Core" Version="0.10.1912.2604-alpha" IsImplicitlyDefined="true" />
+    <PackageReference Condition="$(CsharpGeneration)" PrivateAssets="All"
+      Include="Microsoft.Quantum.CsharpGeneration" Version="0.10.1912.2604-alpha" IsImplicitlyDefined="true" 
+      IsQscReference ="true" ExecutionTarget="Any" Priority="-1" />
+  </ItemGroup>
```

Comment:
Do these need to be added to an update QDK script?

Reply:
There is no such script on the compiler repo. I added them to the script on Solid though.